packages/node/src/evm/fulfillments/api-calls.test.ts

```diff
@@ -227,11 +230,11 @@ describe('submitApiCall', () => {
 
     it('returns an error if everything fails', async () => {
       const provider = new ethers.providers.JsonRpcProvider();
-      const contract = new ethers.Contract('address', ['ABI']);
+      const contract = createAirnodeRrpFake();
       (contract.callStatic.fulfill as jest.Mock).mockRejectedValueOnce(new Error('Server did not respond'));
```

Comment:
@andreogle Any specific reason why mocking the values through the contract variable?
You could just as well do `staticFulfillMock.mockRejectedValueOnce(...)`

Reply:
No reason. I wasn't really sure how to mock ethers for a while and there are a few leftovers from that